### ISSUE TITLE
Local Tokeninfo on port 9021 in not useable from docker containers

### ISSUE THREAD

**Issue by ChristianLohmann** [zalando-stups/taupage#307]
The problem is that if you run  container with standard networking config (`bridge`) it is not possible to access `localhost:9021` from within the container. 
So as a temporary workaround if one wants to use the local tokeninfo he hast to switch the networking environment of the container to `host`.

**Comment by hjacobs** [comment]
Yes, I also realized my mistake after the Taupage release :disappointed:

**Comment by hjacobs** [comment]
We need to replace `localhost` with the Docker bridge IP (similar to what we have for the etcd proxy).
